integrations/pull_merge_test.go

```diff
@@ -61,31 +61,75 @@ func testPullCleanUp(t *testing.T, session *TestSession, user, repo, pullnum str
 
 func TestPullMerge(t *testing.T) {
 	onGiteaRun(t, func(t *testing.T, giteaURL *url.URL) {
-		hookTasks, err := models.HookTasks(1, 1) //Retrieve previous hook number
-		assert.NoError(t, err)
-		hookTasksLenBefore := len(hookTasks)
+		createPullNotified, deferableCreate := notifierListener.RegisterChannel("NotifyNewPullRequest", 0, &models.PullRequest{})
```

Comment:
```suggestion
		createPullNotified, unregisterCreate := notifierListener.RegisterChannel("NotifyNewPullRequest", 0, &models.PullRequest{})
```

I think that `deferableCreate` is confusing. 😕

Reply:
Done